tests/rules/javascript/test.event_listener_fourth.js

```diff
@@ -90,6 +81,9 @@ describe('event_listener_fourth', () => {
       });
   });
```

Comment:
Can't see where in the file event-listener-fourth.js we are not allowing window identifier.
 And why all the other  tests are called on window and don't report this warning but here it's not allowed.
 Also why 'should not allow test' doesn't report any warnings.